src/SFA.DAS.AdminService.Web/Controllers/SearchTrainingProviderController.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace SFA.DAS.AdminService.Web.Controllers
+{
+    [Authorize(Roles = Domain.Roles.TribalTeam)]
```

Comment:
This controller is not supposed to be in here. We need to put a link to web qa instead,